data/net/minecraft/network/protocol/game/ClientboundBlockEventPacket.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/network/protocol/game/ClientboundBlockEventPacket
+	METHOD <init> (Lnet/minecraft/core/BlockPos;Lnet/minecraft/world/level/block/Block;II)V
+		ARG 1 pos
+		ARG 2 block
+		ARG 3 b0
+		ARG 4 b1
```

Comment:
Could b0 and b1 be more descriptive?

Reply:
I've named them `paramA` and `paramB` now, which is what they are called in `BlockEventData`.